lib/data_migrator/offence_code_generator.rb

```diff
@@ -34,6 +34,15 @@ def scheme_10_offence_unique_code
     description.abbreviate +
       modifier.to_s +
       '_' +
-      offence_band.description
+      offence_band.description +
+      gt_10_offence_code_suffix
+  end
+
+  def gt_10_offence_code_suffix
+    if fee_schemes&.first&.version && fee_schemes.first.version > 10
```

Comment:
not a blocker - could be worth using a 
```
fee_scheme = fee_schemes.find { |fs| fs.name.eql?('AGFS') }
if fee_scheme&.version && fee_scheme.version > 10
 ...

```

to be more explicit here and return just the one, fee scheme